src/clj/game/cards/programs.clj

```diff
@@ -545,6 +545,11 @@
                                      :effect (effect (trash (assoc target :seen true))
                                                      (shuffle! :corp :deck))}} card))}]}
 
+   "Kyuban"
+   {:hosting {:req #(and (ice? %) (can-host? %))}
+    :events {:pass-ice {:req (req (= (:cid target) (:cid (:host card))))
+                        :effect (effect (gain-credits 2))}}}
```

Comment:
Might want to specify runner side, otherwise this might give money to corp on An Offer You Can't Refuse runs.